source/engine/graphics/Graphics.cpp

```diff
@@ -171,19 +125,20 @@ namespace nc::graphics
     // Note: All calls below are asynchronous fire-and-forget methods. A maximum of MaxFramesInFlight sets of calls will be running at any given time.
     void Graphics::Draw(const PerFrameRenderState& state)
     {
+        auto* currentFrame = m_frameManager->CurrentFrameContext();
```

Comment:
Try to keep optick macros at the top of functions otherwise stuff doesn't get profiled.